libs/causality/src/lib/CausalAnalysisDashboard/Controls/CausalAnalysisView/CausalAggregateView/CausalAggregateView.tsx

```diff
@@ -30,7 +30,7 @@ export class CausalAggregateView extends React.PureComponent<ICausalAggregateVie
 
   public render(): React.ReactNode {
     const styles = CausalAggregateStyles();
-    this.props.data.global_effects.sort((d1, d2) => d2.point - d1.point);
+    this.props.globalEffects.sort((d1, d2) => d2.point - d1.point);
```

Comment:
this will do a in-place sort which impact the parent component pass this value in, will it cause any issue?
for intense method like sorting here, ideally not put in render function, move to componentDidMount/Update could be better.

Reply:
The sorting seems to be present from even from before. I have created a work item for this ask internally to re-locate the sorting function.